cgi.cfg

```diff
@@ -134,6 +134,23 @@ authorized_contactgroup_for_all_service_commands=
 authorized_for_all_host_commands=thrukadmin
 authorized_contactgroup_for_all_host_commands=
 
+# BROADCAST ACCESS
+# A comma-delimited list of usernames that can manage broadcasts.
+authorized_for_broadcasts=thrukadmin
+authorized_contactgroup_for_broadcasts=
+
+# REPORT ACCESS
+# A comma-delimited list of usernames that have full access to the reporting
+# feature
+authorized_for_reports=thrukadmin
```

Comment:
I'd say we set this to `*` because thats the way it was before.

Reply:
Done. See 3d69a5815cd949ce3dcaef2916aca2a1a57674c3